importer/management/commands/runimport.py

```diff
@@ -44,7 +44,7 @@ def get_api_url(app):
     if app == 'blogs':
         return SCRAPY + 'api/blogs/'
     if app == 'media':
-        return SCRAPY + 'api/media_files?page=349'
```

Comment:
I'm guessing this was leftover from a time when someone was restricting scope to make things run faster? 🤷🏼

Reply:
That's my assumption